vendor/bat-native-ads/src/bat/ads/internal/processors/contextual/text_embedding/text_embedding_processor_unittest.cc

```diff
@@ -0,0 +1,57 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/processors/contextual/text_embedding/text_embedding_processor_util.h"
```

Comment:
shouldn't this file be `text_embedding_processor_util_unittest`?

Reply:
I was planning on also adding unittests for parts of text_embedding_processor.cc as we build functionality out, so it would be larger in scope than just testing the util functions.

Reply:
Each class should have its own set of unit tests, i.e.
```
foo.h
foo.cc
foo_unittest.cc

bar.h
bar.cc
bar_unittest.cc
```
We should not combine tests

Reply:
Okay, changed to text_embedding_processor_util_unittest